### PR TITLE
Fix metrics cron job

### PR DESCRIPTION
**Pull request by KPobeeNorris**
The metrics cron job hasn't been running in K8s on staging, and it
appears that this could be due to a yaml formatting issue.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Prison details update (e.g. slot updates)

## Checklist:

- [ ] [My commit message follows the GDS git style standards we have adopted](https://github.com/alphagov/styleguides/blob/master/git.md).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the deployment repo accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: deploy/staging/cronjob.yaml [hunk 1]
```diff
@@ -13,10 +13,10 @@ spec:
       template:
         spec:
           containers:
-          - name: prison-visits-booking-staff-pvb-metrics-refresh
-            image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/prison-visits-booking/prison-visits-staff:latest
-            imagePullPolicy: Always
-            command: ['sh', '-c', "bundle exec rake pvb:metrics:refresh"]
+            - name: prison-visits-booking-staff-pvb-metrics refresh
```
Comment (rossjones) [inline deploy/staging/cronjob.yaml]: Is the missing hyphen in the name deliberate?

File: deploy/staging/cronjob.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ kind: CronJob
 metadata:
   name: refresh-views
 spec:
-  schedule: "0 0 * * *"
+  schedule: "5 1 * * *"
```
Comment (TobyRet) [inline deploy/staging/cronjob.yaml]: I'm not sure I understand how changing the time the job fires fixes the issue? 😢

File: deploy/staging/cronjob.yaml (hunk 1 above)
Comment (TobyRet) [reply deploy/staging/cronjob.yaml]: 👍

File: deploy/staging/cronjob.yaml (hunk 2 above)
Comment (KPobeeNorris) [reply deploy/staging/cronjob.yaml]: Sorry - this was part of my initial thinking, but will change back now I've realised it's indentation yaml pain

**Pull request merged**